components/SearchInput/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { books } from "../../lib/books.js";
+import BookImage from "../BookImage";
+
+export default function SearchInput() {
+  const [searchQuery, setSearchQuery] = useState("");
+  const [searchResults, setSearchResults] = useState([]);
+
+  const handleInputChange = (e) => {
+    const query = e.target.value;
+    setSearchQuery(query);
+
+    if (query === "") {
+      setSearchResults([]);
+    }
+  };
```

Comment:
Maybe write event instead of just e, so the code is better to read :)
I was told that we should follow the neue Fische curriculum convention by writing function declarations instead of arrow functions, so here is an example of how that would look like:

```
 function handleInputChange(event) {
    const query = event.target.value;
    setSearchQuery(query);

    if (query === "") {
      setSearchResults([]);
    }
  };
```

Reply:
Oh thanks @CorneliusMoe 🙏  I will change it immediately.